src/forecast&highlights/UnitButtons.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+
+const UnitButtons = ({ setUnits }) => {
+  const [ChildUnits, setChildUnits] = useState("metric");
+  const [active, setActive] = useState("button-C");
+
+  useEffect(() => {
```

Comment:
You might not need this effect

https://react.dev/learn/you-might-not-need-an-effect

<img width="1203" alt="CleanShot 2023-08-06 at 02 38 59@2x" src="https://github.com/Jijuzo/Weather-App/assets/55194394/ff2c2438-d109-4049-8c10-2c9568560025">